javascript-objects/main.js

```diff
@@ -0,0 +1,37 @@
+const student = {
+  firstName: 'Julie',
+  lastName: 'Luangpraxay',
+  age: 27,
+};
+
+const fullName = student.firstName + student.lastName;
+console.log('value of fullName:', fullName);
+
+student.livesInIrvine = false;
+student.previousOccupation = 'Social Media Specialist';
+console.log('student.livesInIrvine:', student.livesInIrvine);
+
+console.log('student.previousOccupation:', student.previousOccupation);
+console.log('value of student:', student);
+
+const vehicle = {
+  make: 'McLaren',
+  model: '720S',
+  year: 2022,
+};
```

Comment:
Jeez, when are you taking me for a ride?

Reply:
LOL